ocs_ci/utility/ibmcloud.py

```diff
@@ -203,3 +205,193 @@ def add_deployment_dependencies():
     ]
     for cr in wa_crs:
         run_cmd(f"oc apply -f {cr}")
+
+class IBMCloud(object):
+    """
+    Wrapper for Ibm cloud
+    """
+
+    def restart_nodes(self, nodes, timeout=900, wait=True):
+        """
+    	Reboot the node on IBM Cloud.
+
+    	Args:
+            nodes (list): The worker node instance
+
+        """
+        logger.info(f"restart_nodes:")
+        providerId = nodes[0].get()['spec']['providerID']
+        clusterid = providerId.split("/")[5]
+
+        cmd = f"ibmcloud ks workers --cluster {clusterid} --output json"
+        out = run_cmd(cmd)
+
+        out = json.loads(out)
+        node = out[0]['id']
+
+        cmd = f"ibmcloud ks worker reboot --cluster {clusterid} --worker {node} -f"
+        out = run_cmd(cmd)
+        logger.info(f"Node restart command output: {out}")
+
+    def attach_volume(self, volume, node):
+        """
+        Attach volume to node on IBM Cloud.
+
+	Args:
+            volume (str): volume id.
+            node (str): worker node to attach.
+
+        """
+        logger.info(f"attach_volumes:{node[0].get()['metadata']['labels']['failure-domain.beta.kubernetes.io/zone']}")
+        providerId = node[0].get()['spec']['providerID']
+        clusterid = providerId.split("/")[5]
+        worker_id = node[0].get()['metadata']['labels']['ibm-cloud.kubernetes.io/worker-id']
+
+        logger.info(f"volume is : {volume}")
+
+        cmd = f"ibmcloud ks storage attachment  create --cluster {clusterid} --worker {worker_id} --volume {volume} --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        logger.info(f"attachment command output: {out}")
+
+
+    def detach_volume(self, volume, node=None, delete_from_backend=True):
+        """
+        Detach volume to node on IBM Cloud.
+
+	Args:
+            volume (str): volume id.
+            node (str): worker node to detach.
+	    delete_from_backend (bool): True for deleting the disk from the
+                storage backend, False otherwise
```

Comment:
this next line should be intended only by 4 spaces

Reply:
Fixed